src/hotspot/share/jfr/periodic/jfrThreadCPULoadEvent.cpp

```diff
@@ -132,7 +132,7 @@ void JfrThreadCPULoadEvent::send_events() {
     }
   }
   log_trace(jfr)("Measured CPU usage for %d threads in %.3f milliseconds", number_of_threads,
-    (double)(JfrTicks::now() - event_time).milliseconds());
```

Comment:
I think this one is a simple bug - the wrong format specifier is being used. I don't think this expects to see e.g. 3.6 milliseconds. But confirm with JFR folk.

Reply:
I don't know what the original intent was, but seems fine to print a fraction now,